src/java/org/apache/cassandra/db/filter/ColumnFilter.java

```diff
@@ -924,20 +926,22 @@ public String toString()
             {
                 prefix = queried.statics.isEmpty()
                        ? "<all regulars>/"
-                       : String.format("<all regulars>+%s/", toString(queried.statics.selectOrderIterator(), false));
+                       : String.format("<all regulars>+%s/", toString(queried.statics.selectOrderIterator(), false, false));
             }
 
-            return prefix + toString(queried.selectOrderIterator(), false);
+            return prefix + toString(queried.selectOrderIterator(), false, false);
         }
 
         @Override
-        public String toCQLString()
+        public String toCQLString(boolean redact)
         {
-            return queried.isEmpty() ? "*" : toString(queried.selectOrderIterator(), true);
+            return queried.isEmpty() ? "*" : toString(queried.selectOrderIterator(), true, redact);
         }
 
-        private String toString(Iterator<ColumnMetadata> columns, boolean cql)
+        private String toString(Iterator<ColumnMetadata> columns, boolean cql, boolean redact)
         {
+            assert cql || !redact : "Cannot redact non-CQL representation";
```

Comment:
It's easier to understand the assertion and match to the message if it's:
```suggestion
            assert !cql && redact : "Cannot redact non-CQL representation";
```